_installation/compiling-babelfish-from-source.md

```diff
@@ -6,261 +6,211 @@ nav_order: 2
 
 # Compiling Babelfish from source
 
-This section will walk you through the steps required to create a working Babelfish installation on Linux. 
+This section will walk you through the steps required to create a working Babelfish installation on an Ubuntu 20.04 Linux host. Please note that the steps may vary on other operating systems, but the overall process is roughly the same. 
 
 
 ## Downloading the source code
 
-Before building Babelfish, you will need to download the source code.  Babelfish source code is separated into two repositories that contain:
+Before building Babelfish, you will need to download the source code.  Babelfish source code is stored in the following repositories: FIXME-are_these_still_the_same_repo_names_?
 
 - the [PostgreSQL database engine](https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish) source code, with changes that provide the protocols, language parsers, and features required by Babelfish. 
 - the [extensions](https://github.com/babelfish-for-postgresql/babelfish_extensions) that support the T-SQL protocol, the T-SQL language, the TDS Protocol, and so on.
 
+FIXME - Do we need to ask the user to navigate into a specific directory location before invoking these commands so the build commands later in this page work?
+
 If you have installed git, you can clone the repos with the following commands:
 
 ``` sh
 git clone https://github.com/babelfish-for-postgresql/postgresql_modified_for_babelfish.git
 git clone https://github.com/babelfish-for-postgresql/babelfish_extensions.git
 ```
 
-## Preparing your system
-
-Most of the packages required to compile Babelfish are part of a typical Linux distribution.  You may find that the packages on your distribution use a similar (but not identical) name.  You will also need a user with root privileges, so you can convey privileges with `sudo`.
+## Prerequisites
 
-In addition, we recommend installing the following additional packages on top of the hard requirements listed above:
+Many of the Babelfish prerequisites are part of a typical Linux distribution.  You may find that the packages on your distribution use a similar (but not identical) name.  To build Babelfish, you will need access to a user with root privileges, so you can convey privileges with `sudo`. 
 
-- [Flex 2.6.4](https://github.com/westes/flex)
-- [Libxml2](http://xmlsoft.org/) development libraries
-- [Open SSL](https://www.openssl.org/) development libraries
-- [Readline](https://tiswww.cwru.edu/php/chet/readline/rltop.html) development libraries
-- [Zlib](https://zlib.net/)
-- [OSSP uuid](http://www.ossp.org/pkg/lib/uuid/) development libraries
-- [pkg-config](https://linux.die.net/man/1/pkg-config)
-- [ICU](https://icu.unicode.org/) development libraries
-- [Bison 3.0.5 or higher](https://www.gnu.org/software/bison/)
+FIXME - Do we need to create a user named 'runner'? If so:
```

Comment:
No, we should remove all those references and simply warn the user that Postgres can not be run as root.

Reply:
Those references are gone... I mention creating a non-root user in 30/34, but don't say explicitly why it must be non-root.